java/src/main/java/PerfPure.java

```diff
@@ -324,9 +323,9 @@ private double complexAbs2(double zReal, double zImag) {
 
     protected int mandelperf() {
         int mandel_sum = 0;
-        for (double re=-2.0; re<=0.5; re+=0.1) {
-            for (double im=-1.0; im<=1.0; im+=0.1) {
-                int m = mandel(re,im);
+        for (double re=-20; re<=5; re+=1) {
+            for (double im=-10; im<=10; im+=1) {
+                int m = mandel(re/10,im/10);
```

Comment:
@michaellilltokiwa could you explain this change?

Reply:
I took this change from the go version:

> 	// These loops are constructed as such because mandel is very sensitive to
> 	// its input and this avoids very small floating point issues.

Reply:
Is this required for the assertion fix?

Reply:
Regarding the literal, xref to https://github.com/JuliaLang/julia/pull/13133 by @btracey for the Go version.

In the Go case, it had negligible effects on performance.